src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANCache.java

```diff
@@ -286,10 +300,15 @@ public boolean isCachedRes(String orgName, String pkgName, String resName) throw
             // get the resource and populate the resource map
             JSONObject result = (JSONObject) res.getJsonObject().get("result");
             JSONArray resources = (JSONArray) result.get("resources");
-            LOGGER.debug("Going to populate the resources cache (orgName=" + orgName + ", pkgName=" + pkgName
-                    + ")");
-            populateResourcesMap(resources, orgName, pkgName, true);
-            return true;
+            
+            if (resources.size() == 0) {
+                return false;
+            } else {
+                LOGGER.debug("Going to populate the resources cache (orgName=" + orgName + ", pkgName=" + pkgName
```

Comment:
Love this line! :p "I am going to try to populate... , but I cannot assure it... :p" Just kidding! ;)